examples/fuse_examples/imaging/classification/stoic21/runner_stoic21.py

```diff
@@ -312,6 +322,8 @@ def run_infer(paths: dict, infer_common_params: dict):
 # Eval Template
 ######################################
 def run_eval(paths: dict, eval_common_params: dict):
+    infer_file = os.path.join(paths['inference_dir'], eval_common_params['infer_filename'])
```

Comment:
`infer_file` is being defined twice (also in `run_infer`), we can consider moving it to `PATHS`.

Reply:
You might have few inference files so I prefer to keep them separated